frontend/src/components/teamsAndOrgs/messages.js

```diff
@@ -8,6 +8,10 @@ export default defineMessages({
     id: 'management.messages.notAllowed',
     defaultMessage: 'You are not allowed to manage organizations.',
   },
+  notAllowedPartners: {
```

Comment:
There are messages here that belongs to partners, please move them to a file in `partners/messages.js` with their corresponding id `partners.[...]` then check that all messages are defined and run `yarn build-locales` to update the `locales/es.js` file. 


CC @CristianAmici @dgquail

Reply:
@emi420 I'm getting this error when I try to run `yarn build-locales`:
`Error: ENOENT: no such file or directory, open '.\src\locales\en.json'`
Can you help me out with this?

Reply:
Yes! You're running that command from the _frontend_ directory?

Reply:
yes

Reply:
You can also try manually:

`npx combine-messages -i './src/**/messages.js' -o './src/locales/en.json'`

Reply:
is there:
\frontend\src\locales\en.json

Reply:
Have you tried manually? maybe it's a Windows issue? I see that your slashes are backward `\`

Reply:
I don’t know if it’s my OS, but manually this is the issue:
SyntaxError: ....frontend\src\components\projectEdit\messages.js: [React Intl] Messages must be statically evaluate-able for extraction.
and finally it says BABEL_TRANSFORM_ERROR.

Reply:
Ok, it's working now! this error is about a different thing, we're working on it. 

In the meanwhile please do the following:

Edit [frontend/src/components/projectEdit/messages.js ](https://github.com/hotosm/tasking-manager/blob/develop/frontend/src/components/projectEdit/messages.js)

1. Comment line 3

`// import projectCreateMessages from '../projectCreate/messages';`

2. Change line 444 to:

```
 projectNameValidationError: {
    id: 'management.projects.edit.errors.project_name_validation_error',
    defaultMessage: 'Project name should start with an alphabet.',
 },
 ```
  
This will get ride of that error.

Reply:
ok, now the problem we have is that there are some duplicate IDs between the messages for partners and the rest of the project.

Reply:
Yes, see this: 

https://github.com/hotosm/tasking-manager/pull/6449#pullrequestreview-2111076033

Reply:
done! and run ok.